data/hai/hai reveal 3 early.txt

```diff
@@ -31,7 +31,7 @@ mission "Hai Building Mountaintop [1] Construction Fleet"
 			`When you return to Hai-home, you are directed to a government hangar where you are met by Xilin and Sayari, both looking very relieved.`
 			branch buildingdeferred
 				has "Building Mountaintop Deferred"
-			`	"I am very glad to see you returned, Captain <last>. Sayari said you were away on an important errand, but I've been hoping for a day now you would appear. I think you might be the solution to the current impasse."`
+			`	"I am very glad to see you returned, Captain <last>. Sayari said you were away on an important errand, but I've been hoping for days now you would appear. I think you might be the solution to the current impasse."`
```

Comment:
This one isn't a typo.

Reply:
You make a good point. However, given that the passage of time in the game is measured in days, IMO this makes more sense. If I'm overruled, that's completely fine.

Reply:
That one was intentional. If it had been more than a day it would be expected that you'd have been sent a message.